docs/api-docs/getting-started/webhooks/webhook-events.md

```diff
@@ -59,7 +39,289 @@
 }
 ```
 
-<a id="webhook-events_orders"></a>
+| Name | Definition |
+| -- | -- |
+| `scope` | The [event](/api-docs/getting-started/webhooks/webhook-events) registered when the webhook was created. |
+| `store_id` | A numerical identifier that is unique to each store. |
+| `data` | A lightweight description of the [event](/api-docs/getting-started/webhooks/webhook-events) that triggered the webhook. Will vary depending on the event registered. |
+| `hash` | The payload data json encoded then passed through SH1 encryption. |
+| `created_at` | Unix timestamp of the date the hook was created. |
+| `producer` | Will always follow the pattern `stores/store_hash`. This is the store that created the webhook. |
+
+
+
+## Cart
+
+```json
+{
+    "scope": "store/cart/created",
+    "store_id": "1025646",
+    "data": {
+        "type": "cart",
+        "id": "09346904-4175-44fd-be53-f7e598531b6c"
+    },
+    "hash": "352e4afc6dd3fc85ea26bfdf3f91852604d57528",
+    "created_at": 1561482670,
+    "producer": "stores/{store_hash}"
+}
+```
+
+| Name | Description |
+|-|-|
+| store/cart/* | Subscribe to all cart events. This will also subscribe you to cart/lineItem. |
+| store/cart/created | This webhook will fire whenever a new cart is created, either via a storefront shopper adding their first item to the cart, or when a new cart is created via an API consumer. If it is from the storefront, then it fires when the first product is added to a new session.(The cart did not exist before). For the API it means a `POST` to /carts, (V3 and Storefront API). The `store/cart/updated` hook will also fire.|
+| store/cart/updated | This webhook is fired whenever a cart is modified through the changes in its line items. Eg. when a new item is added to a cart or an existing item’s quantity is updated. This hook also fires when the email is changed during guest checkout or when an existing item is deleted. The payload will include the ID of the cart being updated. <br> This webhook also fires along with the cart created hook, because the first product being added to an empty cart triggers an update. It will also fire for the following events: <br> - Logging into customer account after creating a cart (email is inherited from customer account email) <br>- Entering email address via guest checkout <br> -Changing the email in guest checkout |

```

Comment:
```suggestion
| store/cart/updated | This webhook fires whenever you modify a cart by changing its line items. For example, when you add a new item to a cart or update an existing item’s quantity. This hook also fires when you change the email during guest checkout or when you delete an existing item. The payload will include the ID of the cart being updated. <br> This webhook also fires along with the cart created hook, because the first product being added to an empty cart triggers an update. It will also fire for the following events: <br> - Logging into customer account after creating a cart (email is inherited from customer account email) <br>- Entering email address via guest checkout <br> -Changing the email in guest checkout |

```

Reply:
I tried to remove passive voice without altering the meaning.